composeml/label_times.py

```diff
@@ -378,5 +485,95 @@ def infer_type(self):
 
         if is_discrete:
             return 'discrete'
-        else:
-            return 'continuous'
+
+        return 'continuous'
+
+    def equals(self, other):
+        """Determines if two label time objects are the same.
+
+        Args:
+            other (LabelTimes) : Other label time object for comparison.
+
+        Returns:
+            bool : Whether label time objects are the same.
+        """
+        return super().equals(other) and self.settings == other.settings
+
+    def _load_settings(self, path):
+        """Read the settings in json format from disk.
+
+        Args:
+            path (str) : Directory on disk to read from.
+        """
+        file = os.path.join(path, 'settings.json')
+        assert os.path.exists(file), 'settings not found'
+
+        with open(file, 'r') as file:
+            settings = json.load(file)
+
+        if 'dtypes' in settings:
+            dtypes = settings.pop('dtypes')
+            self = LabelTimes(self.astype(dtypes))
+
+        self.settings.update(settings)
+
+        return self
+
+    def _save_settings(self, path):
+        """Write the settings in json format to disk.
+
+        Args:
+            path (str) : Directory on disk to write to.
+        """
+        dtypes = self.dtypes.astype('str')
+        self.settings['dtypes'] = dtypes.to_dict()
+
+        file = os.path.join(path, 'settings.json')
+        with open(file, 'w') as file:
+            json.dump(self.settings, file)
+            del self.settings['dtypes']
+
+    def to_csv(self, path, filename='label_times.csv', save_settings=True):
+        """Write label times in csv format to disk.
+
+        Args:
+            path (str) : Location on disk to write to (will be created as a directory).
```

Comment:
If the directory already exists, it will only overwrite the files we want to save. Any other files that already exist in the directory will remain the same.